src/app/MenuDesktop.svelte

```diff
@@ -75,7 +75,7 @@
   $: userDisplay = deriveProfileDisplay($pubkey)
 </script>
 
-<div class="fixed bottom-0 left-0 top-0 z-modal w-72 bg-tinted-700 transition-colors">
+<div class="fixed bottom-0 left-0 top-0 z-nav w-72 bg-tinted-700 transition-colors">
```

Comment:
What's the reason for changing this? This breaks the orange overlapping line.

Reply:
It used to be z-nav, I changed it to z-modal to have it above the modal backlight when rewriting the modals.
Changed it back to z-nav again while moving the modal backlight rightward because my new fullscreen image viewer would not get above this sidebar when triggered inside the modal.
Long story short it's a battle of z-index context. At the same time having the nav set to z-nav made sense

Reply:
The right thing to do here is probably to add a new z index or two in tailwind.config.js and use that, since the current state isn't correct.